signing_clients/tests/__init__.py

```diff
@@ -97,9 +102,17 @@ def test_file(fname):
 
 class SigningTest(unittest.TestCase):
 
+    def setUp(self):
+        self.tmpdir = tempfile.mkdtemp()
```

Comment:
also, please prefix this directory like ``tmp-signing-client-`` or something so it's clear on the system